web/app/static/tags/stack/ambarirepositoryconfig.tag

```diff
@@ -1,51 +1,28 @@
-<div class="form-group" name="ambari_stack1">
-    <label class="col-sm-3 control-label" for="ambari_stack">{{msg.cluster_form_ambari_repo_stack_label}}</label>
+<div class="form-group" name="ambari_repo_version_fg">
+    <label class="col-sm-3 control-label" for="ambari_repo_version">{{msg.cluster_form_ambari_repo_baseurl_label}}</label>
```

Comment:
should be cluster_form_ambari_version_label instead of cluster_form_ambari_version_label